docs/benchmarks.md

```diff
@@ -1,131 +1,192 @@
 # GCSFuse Performance Benchmarks
 
-[FIO](https://fio.readthedocs.io/en/latest/) is used to perform load tests on GCSFuse. Below tables shows performance metrics of GCSFuse for different workloads for the given test setup:
+[FIO](https://fio.readthedocs.io/en/latest/) is used to perform load tests on
+GCSFuse. Below tables shows performance metrics of GCSFuse for different
+workloads for the given test setup:
 
 ## Test setup:
 
 * Infra: GCP VM
 * VM Type: n2-standard-96
-* OS:  ubuntu-20.04
+* OS: ubuntu-20.04
 * VM Bandwidth: 100Gbps
 * VM location: us-west1-b
 * Disk Type: SSD persistent disk
 * GCS Bucket location: us-west1
 * Framework: FIO
+* Bucket location: us-west1
```

Comment:
we already have bucket location on line 15

Reply:
Thanks done.